src/database/db_build.js

```diff
@@ -0,0 +1,42 @@
+const client = require('./db_connection.js');
+
+const runDatabaseBuild = () => {
+  client.connect((connectionError) => {
```

Comment:
@ahmedisam99 You can omit the parenthesis if a function only has one argument (if you want).  This is just a personal style thing, either way isn't right or wrong, so it's up to you.
Then it would look like `client.connect( connectionError => {`

Reply:
@tomduggan85 
I hate them to lol, but eslint enforces this style and we must follow it's instructions (we must use it :<)

Reply:
fk off eslint best practices :P

Reply:
Ah yeah.  You could customize your eslint config for the project, but I suppose that's something that everyone has to agree on, so it's probably a project for another day :)